Frameworks/CoreText/CTFont.mm

```diff
@@ -621,12 +621,17 @@ CFArrayRef CTFontCopyFeatureSettings(CTFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 bool CTFontGetGlyphsForCharacters(CTFontRef font, const UniChar characters[], CGGlyph glyphs[], CFIndex count) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    bool succeeded = true;
+    FT_Face face = (FT_Face)[(UIFont*)font _sizingFontHandle];
```

Comment:
We have to only use DWrite apis here - freetype will be removed after we migrate to DWrite.

---
In reply to: [78248109](https://github.com/Microsoft/WinObjC/pull/989#discussion_r78248109) [](ancestors = 78248109)